astor/code_gen.py

```diff
@@ -368,7 +389,7 @@ def visit_withitem(self, node):
         self.conditional_write(' as ', node.optional_vars)
 
     def visit_NameConstant(self, node):
-        self.write(node.value)
+        self.write(str(node.value))
 
```

Comment:
This is an actual bug, that wasn't noticed until I enhanced the string pretty printing.  Without the `str()`, this would leak things like `None` into the result list.